### ISSUE TITLE
Update alternatives

### ISSUE THREAD

**Issue by eliseuegewarth** [eliseuegewarth/ubuntu-setup-script-toolkit#1]
Make alternatives to:
- [ ] Ubuntu 18.04
- [ ] Fedora
- [ ] Linux Mint

**Comment by eliseuegewarth** [comment]
@egewarth Tem alguma distro ou versão que seria interessante ter nas possibilidades?

**Comment by egewarth** [comment]
geralmente uso só o ubuntu

**Comment by egewarth** [comment]
Qual a real necessidade da issue? tornar o script utilizável em outras distros além do ubuntu?

**Comment by eliseuegewarth** [comment]
@egewarth Sim. Em algumas tentativas percebi que algumas dependências atuais não são compatíveis com a versão 18.04 do Ubuntu e o acoplamento atual inviabiliza a instalação em distros que não sejam Gnome na interface.

**Comment by eliseuegewarth** [comment]
O objetivo é modularizar o código e construir setups que sejam compatíveis com algumas versões das distros selecionadas.

**Comment by egewarth** [comment]
Então minha resposta permanece a mesma, só uso ubuntu kkk
